File: todo/Hello.java
public class Hello {
    public static void main(String[] args) {
        System.out.println("Hello from Java!");
        System.out.printf("a + b = %d\n", doAdd(1, 2));
    }

    public static int doAdd(int a, int b) {
        return a + b;
    }
}

File: todo/Hello.kt
// To compile: kotlinc <name>.kt -include-runtime -d <name>.jar
// To run: java -jar <name>.jar 

fun doAdd(a: Int, b: Int): Int {
    return a + b
}

fun main() {
    println("Hello from Kotlin!")
    println("a + b = ${doAdd(1, 2)}")
}
